packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-merge-section.tsx

```diff
@@ -1,25 +1,67 @@
-import { useState } from 'react'
+import { Dispatch, FC, MouseEvent, SetStateAction, useState } from 'react'
 
-import { Accordion, CopyButton, Icon, Layout, StackedList } from '@/components'
+import { Accordion, Button, CopyButton, Icon, Layout, StackedList } from '@/components'
 import { cn } from '@utils/cn'
+import { PanelAccordionShowButton } from '@views/repo/pull-request/details/components/conversation/sections/panel-accordion-show-button'
 import { isEmpty } from 'lodash-es'
 
 import { LineDescription, LineTitle } from './pull-request-line-title'
 
+interface StepInfoProps {
+  item: {
+    step: string
+    description: string
+    code?: string
+    comment?: string
+  }
+  index: number
+}
+
+const StepInfo: FC<StepInfoProps> = ({ item, index }) => {
+  return (
+    <li key={index}>
```

Comment:
Looking at it, I don't think you need a `key` at all here since the loop is done elsewhere.
```suggestion
}

const StepInfo: FC<StepInfoProps> = ({ item }) => {
  return (
    <li>
```